code/game/atoms.dm

```diff
@@ -146,6 +146,8 @@
 
 	to_chat(user, "\icon[src] That's [f_name] [suffix]")
 	to_chat(user, desc)
+	if(description_cult && (user.mind?.special_role == "Cultist" || isobserver(src)))
```

Comment:
Can't this just use description_antag instead?

Reply:
no, wouldn't look good